### PR TITLE
fetch coin gecko id based on coin symbol

### PR DESCRIPTION
**Pull request by ayrat555**
Coin Gecko provides API with all listed coins.
We can use this API to  find coin id instead of setting
it by hand.

## Changelog
- fetch coin gecko id based on the coin symbol

### CODE REVIEW

File: apps/explorer/test/explorer/exchange_rates/source/coin_gecko_test.exs [hunk 1]
```diff
@@ -18,6 +18,26 @@ defmodule Explorer.ExchangeRates.Source.CoinGeckoTest do
   }
   """
 
+  @coins_list """
```
Comment (vbaranov) [inline apps/explorer/test/explorer/exchange_rates/source/coin_gecko_test.exs]: @ayrat555 let's add coins for all chains, that we support, to the tests

Comment (ayrat555) [reply apps/explorer/test/explorer/exchange_rates/source/coin_gecko_test.exs]: @can you please list their coin symbols defined as `COIN` in env vars?

Comment (vbaranov) [reply apps/explorer/test/explorer/exchange_rates/source/coin_gecko_test.exs]: ETH, POA, RBTC, DAI, CLO

@ayrat555 and it seems, the solution doesn't work: zero price and coinmarkectcap on stg.

Comment (ayrat555) [reply apps/explorer/test/explorer/exchange_rates/source/coin_gecko_test.exs]: I added tests for all supported chains

File: apps/explorer/lib/explorer/exchange_rates/source/coin_gecko.ex [hunk 2]
```diff
@@ -44,15 +44,41 @@ defmodule Explorer.ExchangeRates.Source.CoinGecko do
...
+
+        symbol_data =
+          Enum.find(data, fn item ->
+            item["symbol"] == symbol
+          end)
+
+        if symbol_data do
+          {:ok, symbol_data["id"]}
+        else
+          {:errpr, :not_found}
```
Comment (pasqu4le) [inline apps/explorer/lib/explorer/exchange_rates/source/coin_gecko.ex]: `:error` typo

**Comment by ayrat555** [comment]
@vbaranov rootstock market cap calculation uses records from db. I think your db was empty that's why it returned 0. on production it should be correc

**Pull request merged**